server/constants/activities.ts

```diff
@@ -175,14 +183,8 @@ export const ActivitiesPerClass: Record<ActivityClasses, ActivityTypes[]> = {
     ActivityTypes.COLLECTIVE_CONVERSATION_CREATED,
     ActivityTypes.COLLECTIVE_UPDATE_PUBLISHED,
     ActivityTypes.COLLECTIVE_UPDATE_CREATED,
-    ActivityTypes.ACTIVATED_COLLECTIVE_AS_HOST,
-    ActivityTypes.ACTIVATED_COLLECTIVE_AS_INDEPENDENT,
     ActivityTypes.CONVERSATION_COMMENT_CREATED,
     ActivityTypes.UPDATE_COMMENT_CREATED,
-    ActivityTypes.ORGANIZATION_COLLECTIVE_CREATED,
-    ActivityTypes.COLLECTIVE_MEMBER_INVITED,
-    ActivityTypes.USER_CARD_INVITED,
-    ActivityTypes.USER_CARD_CLAIMED,
```

Comment:
These ones are for gift cards, I haven't moved there anywhere but it sounds like things we don't want people to unsubscribe from